Development/nmos/sdp_utils.cpp

```diff
@@ -1121,6 +1159,58 @@ namespace nmos
             }
         }
 
+        // RTP Header Extensions
+        // See https://www.rfc-editor.org/rfc/rfc5285#section-5
+        {
+            auto isExtmap = [](const web::json::value& nv)
+            {
+                return sdp::fields::name(nv) == sdp::attributes::extmap;
+            };
+            auto iter = session_attributes.begin();
+            while ((iter = std::find_if(iter, session_attributes.end(), isExtmap)) != session_attributes.end())
```

Comment:
Would something like this work? (And similar for `sdp_params.hkep` obvs.)

```
sdp_params.extmap = boost::copy_range<std::vector<sdp_parameters::extmap_t>>(session_attributes | boost::adaptors::filtered([](const web::json::value& nv)
{
    return sdp::fields::name(nv) == sdp::attributes::extmap;
}) | boost::adaptors::transformed([](const web::json::value& nv)
{
    return sdp_parameters::extmap_t(...);
}));

Reply:
Perfect! Thank you, didn't notice it in the original code for some reason.